home.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles/home.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <title>Mauritania palms</title>
+</head>
+<body>
+    <header class="header">
+        <img src="images/navBar.svg" alt="navBar" class="navBar">
+        <img src="images/closeIcon.svg" alt="closeIcon" class="closeIcon dis-none">
+        <div class="navUl dis-none" id="navUl">
+            <a href="./home.html" class="home upperCase">home</a>
+            <a href="./about.html" class="about upperCase">about</a>
+        </div>
+        <div class="desktopHeader">
```

Comment:
- [ ] The top dark navbar is missing from your homepage on the desktop version. Kindly ensure to add the aspect in your design

**Your top section:**
![image](https://user-images.githubusercontent.com/12745474/133655514-10c910ea-b5ce-4e94-9201-d6b8209ace8c.png)

**The original top section:**
![image](https://user-images.githubusercontent.com/12745474/133655689-0b18c963-8603-41d0-82b7-d4b63b337373.png)

Reply:
Addressed this